lib/test/cdo/contact_rollups/test_pardot_v2.rb

```diff
@@ -441,7 +441,7 @@ def test_delete_prospects_by_email_does_not_work_in_non_production
 
   def test_delete_prospects_by_email_finds_matching_pardot_ids
     email = 'test@domain.com'
-    read_prospect_url = "#{PardotV2::PROSPECT_READ_URL}/#{email}"
+    read_prospect_url = "#{PardotV2::PROSPECT_READ_URL}/#{URI.encode_www_form_component(email)}"
```

Comment:
See other comment, I think these Pardot prospect read calls need to have emails with "@" (maybe `%40` works?)